### DISCUSSION TITLE
Using FML with Serverpod

### DISCUSSION THREAD

**Discussion by perumalrollfi** [AppDaddy-Software-Solutions-Inc/Flutter-Markup-Language#261] [General]
Hi,

is it possible to use FML with Serverpod?

Thanks
Perumal

**Comment by JeffOlajos** [comment]
Hi Perumal.

We haven’t as of yet attempted that but I’m sure it’s possible. If you like
we can meet via zoom and kick around some ideas. Jeff
On Thu, Mar 9, 2023 at 4:43 PM Perumal Ramakrishnan <
***@***.***> wrote:

> Hi,
>
> is it possible to use FML with Serverpod?
>
> Thanks
> Perumal
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/AppDaddy-Software-Solutions-Inc/Flutter-Markup-Language/discussions/261>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANAZAU2UDZVDKC5FGQX5UULW3JFJTANCNFSM6AAAAAAVVU6RVI>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID:
> <AppDaddy-Software-Solutions-Inc/Flutter-Markup-Language/repo-discussions/261
> @github.com>
>
--

**Reply by perumalrollfi** [reply]
Hi Jeff,

Thanks for the quick response.  That will be great. What will be your convenient time?

Thanks
Perumal

**Comment by JeffOlajos** [comment]
Where about are you? We are located just outside Toronto and in Eastern
time. Whenever works best for you.  I’ll clear my schedule if need be.

On Thu, Mar 9, 2023 at 5:12 PM Perumal Ramakrishnan <
***@***.***> wrote:

> Hi Jeff,
>
> Thanks for the quick response. That will be great. What will be your
> convenient time?
>
> Thanks
> Perumal
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/AppDaddy-Software-Solutions-Inc/Flutter-Markup-Language/discussions/261#discussioncomment-5259916>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANAZAUYAY4JSIETBLZ6JZ7DW3JIWVANCNFSM6AAAAAAVVU6RVI>
> .
> You are receiving this because you commented.Message ID:
> <AppDaddy-Software-Solutions-Inc/Flutter-Markup-Language/repo-discussions/261/comments/5259916
> @github.com>
>
--

**Reply by perumalrollfi** [reply]
I am in Orlando and in Eastern as well. 

I am open today till 7 PM Eastern or tomorrow any time till 3PM Eastern except 11 to11:30 AM Eastern,

**Comment by JeffOlajos** [comment]
How about tomorrow at 2? If that works I’ll setup a zoom meeting and send
you the link in the morning.

On Thu, Mar 9, 2023 at 5:27 PM Perumal Ramakrishnan <
***@***.***> wrote:

> I am in Orlando and in Eastern as well.
>
> I am open today till 7 PM Eastern or tomorrow any time except 11 to11:30
> AM Eastern,
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/AppDaddy-Software-Solutions-Inc/Flutter-Markup-Language/discussions/261#discussioncomment-5260134>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANAZAU3RDHKSL4TW7KP6XYDW3JKOHANCNFSM6AAAAAAVVU6RVI>
> .
> You are receiving this because you commented.Message ID:
> <AppDaddy-Software-Solutions-Inc/Flutter-Markup-Language/repo-discussions/261/comments/5260134
> @github.com>
>
--

**Comment by perumalrollfi** [comment]
2PM works for me.

Thanks

**Comment by JeffOlajos** [comment]
Jeff Olajos is inviting you to a scheduled Zoom meeting.

Topic: ServerPod
Time: Mar 10, 2023 02:00 PM Montreal

Join Zoom Meeting
https://us06web.zoom.us/j/86217717129?pwd=YnIvRXk3ejlNc3lhcDErMlB5MG92dz09

Meeting ID: 862 1771 7129
Passcode: 742043
One tap mobile
+16469313860,,86217717129#,,,,*742043# US
+16694449171,,86217717129#,,,,*742043# US

Dial by your location
        +1 646 931 3860 US
        +1 669 444 9171 US
        +1 669 900 6833 US (San Jose)
        +1 689 278 1000 US
        +1 719 359 4580 US
        +1 929 436 2866 US (New York)
        +1 253 205 0468 US
        +1 253 215 8782 US (Tacoma)
        +1 301 715 8592 US (Washington DC)
        +1 305 224 1968 US
        +1 309 205 3325 US
        +1 312 626 6799 US (Chicago)
        +1 346 248 7799 US (Houston)
        +1 360 209 5623 US
        +1 386 347 5053 US
        +1 507 473 4847 US
        +1 564 217 2000 US
Meeting ID: 862 1771 7129
Passcode: 742043
Find your local number: https://us06web.zoom.us/u/kdy7VWpdoY


On Thu, Mar 9, 2023 at 5:48 PM Perumal Ramakrishnan <
***@***.***> wrote:

> 2PM works for me.
>
> Thanks
>
>
> On Thu, Mar 9, 2023 at 5:42 PM Jeff Olajos ***@***.***> wrote:
>
> > How about tomorrow at 2? If that works I’ll setup a zoom meeting and send
> > you the link in the morning.
> >
> > On Thu, Mar 9, 2023 at 5:27 PM Perumal Ramakrishnan <
> > ***@***.***> wrote:
> >
> > > I am in Orlando and in Eastern as well.
> > >
> > > I am open today till 7 PM Eastern or tomorrow any time except 11
> to11:30
> > > AM Eastern,
> > >
> > > —
> > > Reply to this email directly, view it on GitHub
> > > <
> >
> https://github.com/AppDaddy-Software-Solutions-Inc/Flutter-Markup-Language/discussions/261#discussioncomment-5260134
> > >,
> > > or unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/ANAZAU3RDHKSL4TW7KP6XYDW3JKOHANCNFSM6AAAAAAVVU6RVI
> > >
> > > .
> > > You are receiving this because you commented.Message ID:
> > >
> >
> <AppDaddy-Software-Solutions-Inc/Flutter-Markup-Language/repo-discussions/261/comments/5260134
> > > @github.com>
> > >
> > --
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/AppDaddy-Software-Solutions-Inc/Flutter-Markup-Language/discussions/261#discussioncomment-5260345
> >,
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AWGKBSCNTRTOQ6NVNOCMLITW3JMG5ANCNFSM6AAAAAAVVU6RVI
> >
> > .
> > You are receiving this because you authored the thread.Message ID:
> >
> <AppDaddy-Software-Solutions-Inc/Flutter-Markup-Language/repo-discussions/261/comments/5260345
> > @github.com>
> >
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/AppDaddy-Software-Solutions-Inc/Flutter-Markup-Language/discussions/261#discussioncomment-5260410>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANAZAU4LGIP6NX352FRZTG3W3JM3RANCNFSM6AAAAAAVVU6RVI>
> .
> You are receiving this because you commented.Message ID:
> <AppDaddy-Software-Solutions-Inc/Flutter-Markup-Language/repo-discussions/261/comments/5260410
> @github.com>
>


--
